modules/ROOT/pages/_partials/api-mgr.adoc

```diff
@@ -236,12 +212,20 @@ For example: `api-mgr:api:download-proxy --gatewayVersion: 4.0.1  643404 /tmp/`
 > api-mgr:api:edit [flag] <apiInstanceId>
 ----
 
-Edits the API instance passed in `<apiInstanceId>`. +
-Besides the default `--help` flag, this command also accepts:
+This command edits the API instance passed in `<apiInstanceId>`. If the API instance is currently deployed, this command redeploys the edited API instance. +
```

Comment:
@luciaPardo  are we sure that this is true for non flex apis?

Reply:
@glenn-rodgers-sf We reviewed it with Lucia. 
`If the API instance is currently deployed, this command redeploys the edited API instance` this is only valid for flex gateways.